src/v2/controllers/vreplicaset_controller/trusted/exec_types.rs

```diff
@@ -131,6 +174,19 @@ pub struct VReplicaSetSpec {
 impl VReplicaSetSpec {
     pub spec fn view(&self) -> spec_types::VReplicaSetSpecView;
 
+    #[verifier(external_body)]
+    pub fn default() -> (vreplicaset_spec: VReplicaSetSpec)
+        ensures vreplicaset_spec@ == spec_types::VReplicaSetSpecView::default(),
+    {
+        VReplicaSetSpec {
+            inner: deps_hack::VReplicaSetSpec {
```

Comment:
Why not directly call `inner::default()`? VReplicaSetSpec has Default trait now https://github.com/anvil-verifier/anvil/pull/611/files#diff-4b77f89f66f064750dbbf3d4d2e54e55c175c6b36a5f2375c3ebe835f6f95e3bR275